### DISCUSSION TITLE
Only count inversions if the train travels the whole inversion

### DISCUSSION THREAD

**Discussion by infinite-coffee** [OpenRCT2/OpenRCT2#20427] [Feature requests]
a.k.a give the Cheaty Corkscrew Coaster the stat penalties it deserves

RCT seems to count inversions based on how many times the first car enters the inversion, which allows things like [this microcoaster](https://www.youtube.com/watch?v=e1BS1YRTqfQ) to have decent stats even though it's unlikely to be enjoyable in real life. One possible fix is to only count inversions if the first car both enters and exits the inversion (or at least the half-inversion). This would cause microcoasters to face stat penalties without affecting not-quite-micro designs like Shuttle Loop.

Of course this would be controversial -- some people love spamming microcoasters -- but it may be worth considering nonetheless.

**Comment by victwnd** [comment]
This could be added as an option to enable when making a scenario at best. Also, the micro corkscrew design would just turn into a 4x4 micro instead of a 4x2 micro. Not much change there.
